opus/import/table_schemas/obs_instrument_coiss.json

```diff
@@ -415,6 +415,15 @@
               [   1, "W",   "Wide Angle", "020", "Y", null]
           ]
     },
+    {
+        "field_name": "duration",
+        "field_type": "real8",
+        "pi_category_name": "obs_instrument_coiss",
+        "pi_referred_slug": "duration",
+        "pi_disp_order": 100,
```

Comment:
pi_disp_order needs to be adjusted for all referred slugs to appear after all other disp_orders for this category (we want the referred slugs to be at the bottom of the list.